### ISSUE TITLE
Ensure that commit for issue #311 is removed from 2.1.1 before release but kept in trunk

### ISSUE THREAD

**Issue by mindthegab** [Alfresco/alfresco-sdk#336]
Changes in issues #311 are too big for an SP.

Suggestion is to:

- Branch the current HEAD in a 3.0.x branch
- Remove issue #311 commits from HEAD
- Use HEAD to cut the 2.1.1 release and kick off the testing process as per https://github.com/Alfresco/alfresco-sdk/wiki/SDK-Supported-Features-and-Test-plan

@ohej Assigning to you so that we can start working on it tomorrow and kick the 2.1.1 release off.

**Comment by slanglois** [comment]
My 2 pennies... Wouldn't it be better to create a 2.1.x branch instead?
This leaves the master in its current state, for future investigations and fix of #311, and only touches the branch for the release.
Besides, this branch will allow us to create a 2.1.2 easily in the future, if necessary.

**Comment by ohej** [comment]
I decided to go with Gab's suggestion.

I've created a branch called sdk.next which still contains the commit, then I have reverted commit 6c57d9ede06da87981be711ed23243e3d5b814fc.

When we want it back in master we simply have to revert the revert and the changes are back.

It's safer to revert than reset (and easier) because we don't have to worry about rewriting history, and we don't have to go ind and manually merge the commits after.

Closing the issue.
